### DISCUSSION TITLE
OnlyOffice

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#1031] [Ideas] [answered]
Can I ask what the reason of #1024 is? And why it is getting removed/disabled? I can understand that it will not be extended, but why it is getting removed/disabled?

**Comment by karlitschek** [comment, marked as answer]
The idea with AIO is to provide a nicely preconfigured Nextcloud that works for most people. We don't want to offer choices but good defaults. This is why AIO should offer only Nextcloud Office and not OO or other offices. If someone prefers a different setup or combination of apps then the person has always the freedom to configure it manually as wanted. But this is not in the scope of the totally preconfigured, polished and standard AIO. Hope this makes sense.

**Reply by Zoey2936** [reply]
Yes, this makes sense, thanks for the answer!

**Reply by andreas-traexler** [reply]
Since it should not be included in the AIO interface anymore, are there at least plans to simplify the onlyoffice installation *without* AIO? A "best-practice" kind of way, so to speak, now that the AIO method (which I would call "best practice") is gone?

BR

**Reply by szaimen** [reply]
Please open a new thread for this. Thx!
